common/interpreter/wawaka_wasm/WawakaInterpreter.cpp

```diff
@@ -262,6 +262,11 @@ int32 WawakaInterpreter::evaluate_function(
 // XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX
 void WawakaInterpreter::Finalize(void)
 {
+    // Clear the code buffer
+    if (!binary_code.empty()) {
+        binary_code.clear();
+    }
+
     // Destroy the environment
```

Comment:
hate to be persnickety about this... but do we really need to check for empty? just clear it.